### DISCUSSION TITLE
How to set diff context option permanently in file?

### DISCUSSION THREAD

**Discussion by kfkawalec** [helmfile/helmfile#732] [Questions]
I can set context inline manually, but how to do this automatically for all commands?
For example:
helmfile diff --context 5

**Comment by yxxhero** [comment]
@kfkawalec please provide more info. Thanks very much.

**Comment by kfkawalec** [comment]
Currently "helmfile apply" generates a full diff even when I only change one small thing.
I can use "helmfile apply --context 5" and then I will only see the "essence" of the change.
In hemlfile.yaml I can set "helmDefaults" but I can't add a variable for "context".
I'm looking for ways if/how this can be done in a file hemlfile.yaml

**Comment by yxxhero** [comment]
@kfkawalec no way except cmd args. this is a feature request. did you have the interest to do this?

**Reply by kfkawalec** [reply]
Yes, I want such solution, but I don't have the skills to make a pull request.
